projects/azure/base-images/e2e-test.yaml

```diff
@@ -0,0 +1,51 @@
+kind: List
+apiVersion: v1
+items:
+
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: e2e-test-base
+    namespace: azure
+
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: e2e-test-base
+    namespace: azure
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: e2e-test-base:latest
+    source:
+      dockerfile: |
+        FROM docker.io/centos:7
+        RUN mkdir -p /go && chmod -R 777 /go && yum update -y && \
+            yum install tar wget git make which golang -y 
+        RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc && \
+            sh -c 'echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/yum.repos.d/azure-cli.repo' && \
+            yum install azure-cli -y
+        RUN  wget https://raw.githubusercontent.com/openshift/openshift-ansible/master/images/installer/origin-extra-root/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-PaaS -O /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-PaaS && \
+             wget https://raw.githubusercontent.com/openshift/openshift-ansible/master/images/installer/origin-extra-root/etc/yum.repos.d/centos-openshift-origin.repo -O /etc/yum.repos.d/openshift-origin.repo && \
+             yum install origin-clients -y
```

Comment:
Where do you need oc?

Reply:
In test image/template/target there is placeholder for `which oc`. Just kept the standard template for now.